extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.js

```diff
@@ -0,0 +1,654 @@
+        ></a>
+        <a target="_blank" href="https://amp.dev/about/stories/"
+          >Learn more about Web Stories<svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="32"
+            height="32"
+            viewBox="0 0 32 32"
+            fill="none"
+          >
+            <path
+              d="M15.9998 10.6667L15.0598 11.6067L18.7798 15.3333H10.6665V16.6667H18.7798L15.0598 20.3933L15.9998 21.3333L21.3332 16L15.9998 10.6667Z"
+              fill="white"
+            /></svg
+        ></a>
+        <a
+          target="_blank"
+          href="https://search.google.com/test/amp?url="
+          class="i-amphtml-story-dev-tools-help-search-preview-link"
+          >Web Stories Google Search Preview Tool<svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="32"
+            height="32"
+            viewBox="0 0 32 32"
+            fill="none"
+          >
+            <path
+              d="M15.9998 10.6667L15.0598 11.6067L18.7798 15.3333H10.6665V16.6667H18.7798L15.0598 20.3933L15.9998 21.3333L21.3332 16L15.9998 10.6667Z"
+              fill="white"
+            /></svg
+        ></a>
+        <svg
+          title="cross"
+          class="i-amphtml-story-dev-tools-device-popup-close"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 24 24"
+          fill="white"
+          width="18px"
+          height="18px"
+        >
+          <path
+            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
+          />
+        </svg>
+      </div>
+    </div>
+  `;
+};
+
+/**
+ * Generates the template for the add device dialog.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+const buildAddDevicePopupTemplate = (element) => {
+  const html = htmlFor(element);
+  return html`
+    <div class="i-amphtml-story-dev-tools-device-popup-bg">
+      <div
+        class="i-amphtml-story-dev-tools-device-popup-container i-amphtml-story-dev-tools-device-popup-add-devices"
+      >
+        <h1>Mobile</h1>
+        <div class="i-amphtml-story-dev-tools-device-popup-mobile"></div>
+        <h1>Tablet</h1>
+        <div class="i-amphtml-story-dev-tools-device-popup-tablet"></div>
+        <h1>Desktop</h1>
+        <div class="i-amphtml-story-dev-tools-device-popup-desktop"></div>
+        <svg
+          title="cross"
+          class="i-amphtml-story-dev-tools-device-popup-close"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 24 24"
+          fill="white"
+          width="18px"
+          height="18px"
+        >
+          <path
+            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
+          />
+        </svg>
+      </div>
+    </div>
+  `;
+};
+
+const DEFAULT_DEVICES = 'iphone11discover;oneplus5t;pixel2';
+
+const ALL_SCREEN_SIZES = [
+  {
+    'name': 'Pixel 2',
+    'width': 411,
+    'height': 605,
+    'deviceHeight': 731,
```

Comment:
Can you add a simple code comment to explain the delta between height/deviceHeight?